mlflow/store/tracking/dbmodels/models.py

```diff
@@ -753,3 +758,258 @@ class SqlTraceRequestMetadata(Base):
         PrimaryKeyConstraint("request_id", "key", name="trace_request_metadata_pk"),
         Index(f"index_{__tablename__}_request_id"),
     )
+
+
+class SqlLoggedModel(Base):
```

Comment:
@harupy We'll want to support soft deletion of logged models in the near future as well. Can we add lifecycle stage or comparable state, similar to runs, to support this?

Reply:
sure, will add it.